I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 9, in script
    init python:
  File "game/script.rpy", line 12, in <module>
    from pythoncode.game import Game
IndentationError: unindent does not match any outer indentation level (girls.py, line 1764)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 9, in script
    init python:
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/ast.py", line 805, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/python.py", line 1460, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 12, in <module>
    from pythoncode.game import Game
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/loader.py", line 693, in load_module
    exec code in mod.__dict__
  File "pythoncode/game.py", line 7, in <module>
  File "/home/g2/Games/DefilerWings/DefilerWings-1.4.1-all-noteat2/renpy/loader.py", line 692, in load_module
    code = compile(source, filename, 'exec')
IndentationError: unindent does not match any outer indentation level (girls.py, line 1764)

Linux-6.8.0-83-generic-x86_64-with-debian-trixie-sid
Ren'Py 6.99.7.858
Defiler Wings 1.4.1.fdsc
